requirements.txt

```diff
@@ -1,7 +1,9 @@
-scipy==1.2.1
```

Comment:
This is a simple edit to the requirements for installation, which accounts for the event that users may have versions of the dependencies ahead of what is required.